src/dbaas/logs/alert/alert.iceberg.service.js

```diff
@@ -0,0 +1,17 @@
+angular.module("ovh-api-services").service("OvhApiDbaasLogsAlertIceberg", function (iceberg) {
+    "use strict";
+
+    var alertResource = iceberg("/dbaas/logs/:serviceName/output/graylog/stream/:streamId/alert/:alertId", {
+        serviceName: "@serviceName",
+        streamId: "@streamId",
+        alertId: "@alertId"
+    }, {
+        query: { method: "GET", isArray: true },
+        get: { method: "GET" },
+        post: { method: "POST" },
+        put: { method: "PUT" },
+        "delete": { method: "DELETE" }
```

Comment:
If you don't override AngularJS `$resource` default configuration, those are not needed https://docs.angularjs.org/api/ngResource/service/$resource#$resource-returns